doc/script_commands.txt

```diff
@@ -9031,16 +9031,21 @@ This command is used in all the item scripts for taming items. Running this
 command will make the pet catching cursor appear on the client connected to the
 invoking character, usable on the monsters with the specified pet ID number. It
 will still work outside an item script.
+If the <pet id> is PET_CATCH_UNIVERSAL it will return the monster target ID
+as long as it is in the pet_db.txt and does not have the MD_STATUS_IMMUNE monster mode.
```

Comment:
I'd just say pet database instead of pet_db.txt here. Who knows how long it will be a .txt file...